cubewalkers/experiment.py

```diff
@@ -6,11 +6,12 @@
 
 
 class Experiment:
-    """Stores information from user-specified experimental inputs."""
+    """
+    Stores information from user-specified experimental inputs.
+    """
 
     def __init__(self, override_string: str, comment_char: str = "#") -> None:
-        """_summary_
-
+        """
         Parameters
         ----------
         override_string : str
```

Comment:
- or -
looks a bit weird in the documentation

Reply:
I've fixed this locally. I'll include it with the other changes.